packages/db/schema.prisma

```diff
@@ -419,3 +429,25 @@ model BetaTokens {
 
   @@index([userId])
 }
+
+model Notification {
+  id               Int              @id @default(autoincrement())
```

Comment:
Now that I think of it, should we add
`isRead      Boolean`
`/// Redirects to this url on notification click`
`url?       String`
??

Reply:
ah shit thought i added that

Reply:
we'll derive the url from the foreign key fields i think?

Reply:
I think storing the URL in the table would be useful in situations where the notification type alone is insufficient to generate the URL, such as system notifications, news, external tutorials, blogs, contests, videos, etc. Additionally, it allows for cases where we may want to override the action URL.

Reply:
Probably for the specified categories, yes. But we might need it for `MISC` notifications.

Reply:
That's why I suggested it to be optional `url?`

Reply:
ok ill add it